recipes/meta-apo/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+mkdir -p ${PREFIX}/bin
+
+make 
+
+cp -r bin/* ${PREFIX}/bin
+cp -r example ${PREFIX}
```

Comment:
It is preferable to distribute example data outside conda. Maybe include a download function to the software.